docs/user_guide/template_substitution.md

```diff
@@ -15,3 +15,7 @@ That is, variables of the form `$FOO` or `${FOO}` are supported, and variables
 with `${FOO:default}` a default value will evaluate to their default if not
 specified on the command line. It is an error to specify a `${FOO}` style
 without a default; to make the default an empty string, use `${FOO:}`.
+
+You can also declare variable substitutions in a separate file which is then included in the build command as in this example:
+
+`stacker build --substitute-file <filename>`
```

Comment:
is there an example of the file format somewhere that we could link to here? or just show an example directly here?

Reply:
Added simple example

Reply:
great, thanks